Source/Widgets/BaseLayerPicker/createDefaultImageryProviderViewModels.js

```diff
@@ -59,6 +61,39 @@ define([
         }));
 
         providerViewModels.push(new ProviderViewModel({
+            name: 'Mapbox Satellite',
+            tooltip: 'Mapbox satellite imagery https://www.mapbox.com/maps/',
+            iconUrl: buildModuleUrl('Widgets/Images/ImageryProviders/mapboxSatellite.png'),
+            creationFunction: function() {
+                return new MapboxImageryProvider({
+                    mapId: 'mapbox.satellite'
+                });
+            }
+        }));
+
+        providerViewModels.push(new ProviderViewModel({
+            name: 'Mapbox Streets',
+            tooltip: 'Mapbox streets imagery https://www.mapbox.com/maps/',
+            iconUrl: buildModuleUrl('Widgets/Images/ImageryProviders/mapboxTerrain.png'),
+            creationFunction: function() {
+                return new MapboxImageryProvider({
+                    mapId: 'mapbox.streets'
+                });
+            }
+        }));
+
+        providerViewModels.push(new ProviderViewModel({
+            name: 'Mapbox Satellite',
```

Comment:
Shouldn't this be `Mapbox Streets Basic`?  Is there a standard imagery set for Mapbox terrain?